### ISSUE TITLE
Utility Storage, Home Storage and Solar Roof

### ISSUE THREAD

**Issue by KeithTimimi** [TashaARK/ARK-Invest-Tesla-Valuation-Model#15]
Is this model for Tesla's automotive business only, or the entirety of the enterprise?

If the latter, than utility storage in the form of the Power Pack, which has been previously described as being as big an opportunity as automotive, Powerwall home storage, and Solar Roof should also be factored into the model?

**Comment by jasonpiepmeier** [comment]
I agree, would like to see the energy/solar businesses in model as well.  That could be big upside.  this looks to just be automotive and autonomous MaaS per labels in rows 8 and 23.  @TashaARK please consider releasing a model with the energy/solar businesses as well.  thanks

**Comment by angelmarauder** [comment]
As addition to this train of thought, I would like to see valuation of highway autonomous Tesla Class-8's which begin production next year.
Compare
https://www.tesla.com/semi
with
https://www.thetruckersreport.com/infographics/cost-of-trucking/

**Comment by angelmarauder** [comment]
"ACT forecasts 2019 U.S. Class 8 retail sales at 257,000 units, up from what Ward’s reported as 250,545 in 2018."
https://www.ttnews.com/articles/class-8-sales-jump-397-january

**Comment by Stevenfuzzy** [comment]
I agree. Elon has stated that Class 8 sales are expected to be 100,000/yr by now.  These are commercial customers that will adopt based on ROI and the ROI is amazing.  Reduced liability from currently available autopilot (I've seen 2 bad accidents in the last 2 days where a class 8 couldn't stop and crushed the cars in front of it on the highway).  If/When these trucks start to roll, the safety mandates in the industry will change. 

Power generation and storage will be part of the sales with many class 8s.  Again, large companies will see the ROI of installing Megapacks at destinations so that drivers can bypass the truck stop.  Think Walmart truck unloading and getting charged at the same time.  Charge will likely come from a megapack to decrease electric load during peak hours.  When solar panels hit mass production, it is going to be the commercial companies (hotels, retailers, basically anywhere that has a big roof and large electric demand) that purchase panels in large quantities (like all my stores need solar, not just a house here and there).  Tesla is an energy storage company!  The cars are only the beginning of what reliable portable  electric power can provide.

**Comment by SpaceMonkeyForever** [comment]
I know they are not modelling solar and storage but how are they accounting for it in their valuation? is this valuation ONLY for automotive and ride-sharing?
